packages/components/cascader-panel/src/index.vue

```diff
@@ -127,8 +128,8 @@ export default defineComponent({
         }
       }
 
-      oldConfig = configTemp
-      oldOptions = options
+      oldConfig = cloneDeep(configTemp)
+      oldOptions = cloneDeep(options)
```

Comment:
This fix make no sense and will reproduce #3350

Reply:
what did you mean "will reproduce #3350"?
![image](https://user-images.githubusercontent.com/21104054/134618146-f47adac0-cde0-40f7-b5eb-f860cddf8eaf.png)
![image](https://user-images.githubusercontent.com/21104054/134618176-888ef470-6313-4db0-9f5b-27801e3e6c14.png)

Reply:
ok,i will check it